docs/configuration.md

```diff
@@ -0,0 +1,60 @@
+# Configuration
+
+Operator configuration can be modified by editing the SSP Custom Resource (CR).
+
+## SSP Custom Resource (CR)
+
+To activate the operator, you will need to create a Custom Resource (CR).
+An example CR can be found in the file [config/samples/ssp_v1beta2_ssp.yaml](../config/samples/ssp_v1beta2_ssp.yaml).
+
+## Sample Explained
+
+```
+apiVersion: ssp.kubevirt.io/v1beta1
+kind: SSP
+metadata:
+  annotations:
+    ssp.kubevirt.io/vm-console-proxy-enabled: "true" # Used by VM console proxy operand
+    ssp.kubevirt.io/vm-console-proxy-namespace: "kubevirt" # Used by VM console proxy operand
+    kubevirt.io/operator.paused: "false" # Pause operator reconciliation
```

Comment:
Is possible to opt out this annotation from this example? I can imagine someone will copy it and then the user will wonder why it is not working :D Or at least add some note, that this annotation is only for debugging and will stop reconciliation

Reply:
Done.